test/unit/org/apache/cassandra/index/sai/metrics/QueryMetricsTest.java

```diff
@@ -739,6 +741,62 @@ public void testQueryPlannerMetrics()
         assertTrue(newLogSelectivityEstimated > oldLogSelectivityEstimated);
     }
 
+    @Test
+    public void testDisableQueryPlanMetrics()
+    {
+        CassandraRelevantProperties.SAI_QUERY_PLAN_METRICS_ENABLED.setBoolean(false);
```

Comment:
We need to ensure we reset to default value at the end of the test

Reply:
Fixed